lambda_functions/crud/model.py

```diff
@@ -21,6 +21,8 @@ class Item(Base):
     sentiments = relationship("ItemSentiment", back_populates="item")
     keyphrases = relationship("ItemKeyphrase", back_populates="item")
 
+    reviews = relationship("Review", backref="item")
```

Comment:
What does backref do? Do we need it? For submissions and factchecks we don't have it.

Reply:
With backref, the child-parent relationship is created implicitly, you only have to define it for parent-child